tools/update-zuliprc-api-field

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+"""
+This tool is for updating API key field `zuliprc` files of dummy users
+in development server.
+Usage:
+./tools/update-zuliprc-api-field /path/to/zuliprc_dev
+"""
+
+import argparse
+import configparser
+import subprocess
+import os
+
+if __name__ == '__main__':
+
```

Comment:
We generally don't do this sort of blank line at start of a block.

Reply:
I also don't think it makes sense for this to have that conditional at all; it's not a `.py` file so can't be imported anyway (and I doubt we'll want it to be), and so the indent here is just clutter.